alchemy_mock/comparison.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, print_function, unicode_literals
-import collections
+import collections.abc
```

Comment:
The library still support Python 2. Mapping is available through collections. So the fix would be to import Mapping from collections on Python 2 and collections.abc to be used for Python 3 as mentioned in the bug ticket.

```
try:
    # try collections.abc for Python 3.4+
    from collections.abc import Mapping
except ImportError:
    # fallback to collections for Python 2 compatibility.
    from collections import Maping
```

six already supports this move as `from six.moves.collections_abc import Mapping` from https://github.com/benjaminp/six/commit/0dc54ead62c7fb06edabf1ff22fb71c0c6d958e1 (six 1.13.0)

Reply:
Thank you.